### ISSUE TITLE
Plugin stops working after a while on ST3

### ISSUE THREAD

**Issue by galli-a** [SublimeText/ElasticTabstops#8]
After a while the plugin stops working. Sometimes quitting and restarting ST would restore functionality, but not always. I have not been able to reproduce the issue reliably. The last time it happened, I got this errors in the console:

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 271, in on_modified
    callback.on_modified(v)
  File "elastic_tabstops in /Users/andrea/Library/Application Support/Sublime Text 3/Installed Packages/ElasticTabstops.sublime-package", line 186, in on_modified
  File "elastic_tabstops in /Users/andrea/Library/Application Support/Sublime Text 3/Installed Packages/ElasticTabstops.sublime-package", line 174, in fix_view
AttributeError: 'NoneType' object has no attribute 'id'
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 287, in on_selection_modified
    callback.on_selection_modified(v)
  File "elastic_tabstops in /Users/andrea/Library/Application Support/Sublime Text 3/Installed Packages/ElasticTabstops.sublime-package", line 213, in on_selection_modified
  File "elastic_tabstops in /Users/andrea/Library/Application Support/Sublime Text 3/Installed Packages/ElasticTabstops.sublime-package", line 174, in fix_view
AttributeError: 'NoneType' object has no attribute 'id'
```

**Comment by adzenith** [comment]
That message shouldn't break the plugin permanently... It should theoretically just not work the once. Have you still been seeing this problem?

**Comment by galli-a** [comment]
That message is repeated for every character I insert or delete in the document. The only sure way to make the plugin work again is to uninstall and then reinstall it.

**Comment by adzenith** [comment]
Well, that's exciting. Let me take another look.

**Comment by adzenith** [comment]
This can only happen in two cases:
1. There is no active view.
2. There is no view at all.

So, uh... I'm thinking it's a bug in Sublime. I'm going to try to work around it, I guess.

**Comment by adzenith** [comment]
I guess reopen if it's not fixed? Thanks!

**Comment by galli-a** [comment]
Maybe this trace will help?

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 543, in run_
    return self.run(edit, **args)
TypeError: run() got an unexpected keyword argument 'name'
```

Sorry if my previous messages were misleading. This is the error I see for each key press, not the other one.

**Comment by adzenith** [comment]
Huh... I'm not actually sure if that's from ElasticTabstops. Do your other plugins still work?

**Comment by galli-a** [comment]
I'm not really sure anymore, now that you say that.
I'll try to uninstall all plugins and reinstall one at a time to see which one is the culprit then.
I'll get back to you as soon as I find out

**Comment by adzenith** [comment]
Cool, thanks!

**Comment by galli-a** [comment]
I think I figured it out. The error I saw, and that prevented ElasticTabstops from working, only happens when there also is the SidebarEnhancements plugin installed. No other combination of plugins I tested gave rise to the same error. I'm not really sure what's happening, and if there any other test I can do to help.

**Comment by adzenith** [comment]
It appears we have conflicting copies of Edit.py.

**Comment by adzenith** [comment]
Should be fixed in ab114b978fcc236e0089af77cf1873c9ef177545.

**Comment by adzenith** [comment]
Thanks for hunting down that conflicting plugin, by the way!
